src/main/java/com/extendedclip/papi/expansion/player/PlayerUtil.java

```diff
@@ -199,7 +199,23 @@ public static ItemStack itemInHand(Player p) {
     }
 
     public static int durability(ItemStack item) {
-        return item != null ? item.getType().getMaxDurability() - item.getDurability() : 0;
+        if (item == null) {
+            return 0;
+        }
+
+        if (!VersionHelper.IS_1_21_OR_NEWER) {
+            return item.getType().getMaxDurability() - item.getDurability();
+        }
+
+        final Damageable meta = (Damageable) item.getItemMeta();
```

Comment:
Isn't it unsafe to do such a cast here without checking if it actually is of type Damagable?

Reply:
As far as I can tell, All ItemMeta implements Damageable https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/browse/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaItem.java#172

Spigot is also doing the same in ItemStack#getDurability
![image](https://github.com/user-attachments/assets/9cdfa6bc-5147-4cf0-b165-175dcb425e71)